.travis.yml

```diff
@@ -1,6 +1,11 @@
 sudo: required
 dist: trusty
 language: python
+cache:
+  directories:
+    - $HOME/miniconda
+before_cache:
```

Comment:
Does this delete the packages before the cache? Shouldn't we keep them?

Reply:
No, the cache itself also has to be downloaded and uploaded from S3. The packages installed in the environment are kept, as the are installed with `--copy`.